Translations/pt_BR.po

```diff
@@ -875,359 +871,306 @@ msgstr ""
 msgid "Portal to"
 msgstr "Portal para"
 
-#: Source/cursor.cpp:231
+#: Source/cursor.cpp:238
 msgid "The Unholy Altar"
 msgstr "O Altar Profano"
 
-#: Source/cursor.cpp:233
+#: Source/cursor.cpp:240
 msgid "level 15"
 msgstr "o nível 15"
 
-#: Source/diablo.cpp:114
+#: Source/diablo.cpp:116
 msgid "I need help! Come Here!"
 msgstr "Preciso de ajuda! Venha aqui!"
 
-#: Source/diablo.cpp:115
+#: Source/diablo.cpp:117
 msgid "Follow me."
 msgstr "Siga-me."
 
-#: Source/diablo.cpp:116
+#: Source/diablo.cpp:118
 msgid "Here's something for you."
 msgstr "Tenho algo pra você."
 
-#: Source/diablo.cpp:117
+#: Source/diablo.cpp:119
 msgid "Now you DIE!"
 msgstr "Agora você MORRE!"
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:781
+#: Source/diablo.cpp:787
 msgid "Options:\n"
 msgstr "Opções:\n"
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:782
+#: Source/diablo.cpp:788
 msgid "Print this message and exit"
 msgstr "Imprime esta mensagem e sai"
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:783
+#: Source/diablo.cpp:789
 msgid "Print the version and exit"
 msgstr "Imprime a versão e sai"
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:784
+#: Source/diablo.cpp:790
 msgid "Specify the folder of diabdat.mpq"
 msgstr "Especifique a pasta de diabdat.mpq"
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:785
+#: Source/diablo.cpp:791
 msgid "Specify the folder of save files"
 msgstr "Especifique a pasta dos arquivos salvos"
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:786
+#: Source/diablo.cpp:792
 msgid "Specify the location of diablo.ini"
 msgstr "Especifique o local do diablo.ini"
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:787
+#: Source/diablo.cpp:793
 msgid "Skip startup videos"
 msgstr "Pular vídeos de abertura"
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:788
+#: Source/diablo.cpp:794
 msgid "Display frames per second"
 msgstr "Exibir quadros por segundo"
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:789
-msgid "Run in windowed mode"
-msgstr "Executar no modo janela"
-
-#. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:790
+#: Source/diablo.cpp:795
 msgid "Enable verbose logging"
 msgstr "Habilitar relatório detalhado"
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:791
+#: Source/diablo.cpp:796
 msgid "Record a demo file"
 msgstr "Grava um arquivo de demonstração"
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:792
+#: Source/diablo.cpp:797
 msgid "Play a demo file"
 msgstr "Toca um arquivo de demonstração"
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:793
+#: Source/diablo.cpp:798
 msgid "Disable all frame limiting during demo playback"
 msgstr "Desabilita todos os limites de frame durante a demonstração"
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:794
+#: Source/diablo.cpp:799
 msgid ""
 "\n"
 "Game selection:\n"
 msgstr ""
+"\n"
+"Selecionar jogo:\n"
```

Comment:
```suggestion
"Seleção do jogo:\n"
```
Having said that, I'd also change the original text on this one to match your translation ("Select game").